src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/Symmetric/SymmetricOneShotBase.cs

```diff
@@ -408,6 +409,33 @@ public void DerivedTypesDefineTest()
             Assert.Empty(missingMethods);
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindows7))]
+        public void DeryptOneShot_Cfb8_ToleratesExtraPadding()
+        {
+            using (SymmetricAlgorithm alg = CreateAlgorithm())
+            {
+                if (alg is RC2)
+                {
+                    throw new SkipTestException("RC2 does not support CFB.");
+                }
+
+                alg.Key = Key;
+
+                // This is a plaintext that is manually padded. Though it is going to be
+                // encrypted with CFB8, it's padded to BlockSizeInBytes-1.
```

Comment:
By writing BlockSizeInBytes-1 and having 1 semantic byte, we're really manually padding to the block size.... which I assume is what you meant.

At least, to me, "padded to BlockSizeInBytes-1" would be "we turned a one byte message into a 15 byte message" (for AES.  7 for DES)

Reply:
Ah okay, yeah I see how that was not clear. It was 1 byte of data with N-1 bytes of padding, to make a block-size worth of data. Hopefully that is more clear.